SYCL/Basic/accessor/accessor.cpp

```diff
@@ -1051,5 +1051,36 @@ int main() {
       assert(v[i] == ((i * 2 + 1) + i));
   }
 
+  // Assignment operator test for 0-dim buffer accessor
+  {
+    using AccT = sycl::accessor<int, 0>;
+
+    sycl::queue Queue;
+    int Data = 32;
+    sycl::buffer<int, 1> DataBuffer(&Data, sycl::range<1>(1));
+
+    Queue.submit([&](sycl::handler &CGH) {
+      AccT Acc(DataBuffer, CGH);
+      CGH.single_task<class acc_0_dim_assignment>([=]() { Acc = 64; });
+    });
+    Queue.wait();
+    assert(Data == 64);
+  }
+
+  // Assignment operator test for 0-dim local accessor
+  {
+    using LocalAccT = sycl::local_accessor<int, 0>;
+
+    sycl::queue Queue;
+
+    Queue.submit([&](sycl::handler &CGH) {
+      LocalAccT LocalAcc(CGH);
+      CGH.single_task<class local_acc_0_dim_assignment>([=]() {
+        LocalAcc = 64;
+        assert(LocalAcc == 64);
```

Comment:
SYCL spec does not guarantee that `assert` will work in a kernel. Therefore, I would suggest to storing `LocalAcc` value into some other non-local accessor to check it on host

Reply:
I should have now addressed this in https://github.com/intel/llvm-test-suite/pull/1570/commits/fe9cf992aa8236b5d4dfa8dc8790d0314f4bc928